### ISSUE TITLE
Code Table Request - Additional fixatives

### ISSUE THREAD

**Issue by kmkocot** [ArctosDB/arctos#3845]
I am uploading parts to Arctos from a recent marine invertebrate sampling expedition to Antarctica. We heavily use some atypical fixatives that I would like to have added to the PART_PRESERVATION table.

Here are the preservations and their descriptions. All of these should allow Tissueness.
ethanol, 100%, mol biol grade - Molecular biology grade absolute ethyl alcohol
ethanol, 95%, borax - a solution that consits of 95% ethyl alcohol and 5% water buffered with Borax
ethanol, 95%, borax, glycerin - a solution that consits of 95% ethyl alcohol and 5% water buffered with Borax and with glycerin added to reduce specimen brittleness
ethanol, 95%, glycerin - a solution that consits of 95% ethyl alcohol and 5% water with glycerin added to reduce specimen brittleness

Thank you!
Kevin

**Comment by Jegelewicz** [comment]
I suggest a few tweaks:

ethanol, mol biol grade - Molecular biology grade absolute ethyl alcohol
ethanol, 95% borax buffered - a solution that consists of 95% ethyl alcohol and 5% water buffered with Borax
ethanol, 95% borax buffered with glycerin - a solution that consists of 95% ethyl alcohol and 5% water buffered with Borax and with glycerin added to reduce specimen brittleness
ethanol, 95% with glycerin - a solution that consists of 95% ethyl alcohol and 5% water with glycerin added to reduce specimen brittleness

I am not sure about the "with" part, but using commas to separate concentration AND other stuff in solution will possibly make things weird when this goes out to aggregators. I also considered replacing the "with" with " | ". But maybe we could do this even better? @dustymc @ArctosDB/arctos-code-table-administrators thoughts?

**Comment by dustymc** [comment]
https://arctos.database.museum/info/ctDocumentation.cfm?table=ctpart_preservation

>Molecular biology grade absolute ethyl alcohol

"ethanol, 100%" seems sufficient and consistent - it's all ethanol or it ain't, no?

Avoiding commas (and maybe even updating existing data to do so) seems worthwhile, if not a huge priority at the moment. Pipes don't seem like a great solution, but they're not "native delimiters" for a bunch of things either.

I think we may be trying too hard with the concentration. That 100% ethanol isn't as soon as the jar is opened, and then things get much worse when the first soggy rat get plunked in. Container environment provides a place to record measured concentration (and watch it change over time), but I'm not sure how that lines up with reality or tissues. I don't have any real suggestions, but this seems like something that could benefit from focused discussion.

If any of these are "standard" maybe they could be shortened and elaborated in the documentation - eg "ethanol/borax/glycerin" or similar plus a more-verbose description. (If there are many ways to make that then this probably won't work.)

???????????

**Comment by Jegelewicz** [comment]
>     Molecular biology grade absolute ethyl alcohol
> 
> "ethanol, 100%" seems sufficient and consistent - it's all ethanol or it ain't, no?

I was always told that this was impossible - there is always some amount of water, which is why I opted for "ethanol, mol biol grade"

But I'm no expert!

**Comment by kmkocot** [comment]
Hi @Jegelewicz and @dustymc,

Thanks for your input here! I totally understand the comment about 100% ethanol promptly absorbing water from the air when the bottle is opened, but we pre-fill tubes at the beginning of the cruise and then are putting mostly tiny (ca. 0.5-2 mm) worms into a 2 ml tube with this solution. In practice, using 100% ethanol pretty much rules out histology whereas the small amount of water in 95% ethanol means that we can generally still see gross morphological characters so the difference isn't trivial for us. So, I would really like to see an addition to the part_preservation table related to 100% ethanol if you are willing. 
I like the suggestion of "ethanol, mol biol grade - Molecular biology grade absolute ethyl alcohol" but "ethanol, 100% - a solution that was supplied as 100% ethanol (not considering moisture absorbed from the air)" or something along those lines could be a good compromise?

We can note the molecular biology grade part in the remarks though. On that note, it's also easy for us to just note that borax and/or glycerin were added in the remarks too. The actual preservative is the ethanol so on second thought, I think there's no need to complicate things by creating new additions to the table for those.

I used the syntax with commas to match what is currently in the part_preservation code table. I don't have strong feelings.

Thanks so much!
Kevin
